src/main.py

```diff
@@ -32,26 +32,25 @@ def load_preferences(self):
     def connect_actions(self):
         self.create_action('quit', self.on_quit_action, ['<primary>q'])
         self.create_action('about', self.on_about_action)
-        self.create_action('mode_none', self.set_mode, ['<shift>N', 'F1'], InteractionMode.NONE)
-        self.create_action('mode_pan', self.set_mode, ['<shift>P', 'F2'], InteractionMode.PAN)
-        self.create_action('mode_zoom', self.set_mode, ['<shift>Z', 'F3'], InteractionMode.ZOOM)
-        self.create_action('mode_select', self.set_mode, ['<shift>S', 'F4'], InteractionMode.SELECT)
+        self.create_action('mode_pan', self.set_mode, ['<shift>P', 'F1'], InteractionMode.PAN)
+        self.create_action('mode_zoom', self.set_mode, ['<shift>Z', 'F2'], InteractionMode.ZOOM)
+        self.create_action('mode_select', self.set_mode, ['<shift>S', 'F3'], InteractionMode.SELECT)
         self.create_action('preferences', preferences.open_preferences_window, ['<primary>p'], self)
         self.create_action('plot_settings', plot_settings.open_plot_settings, ["<primary><shift>P"], self)
         self.create_action('add_data', graphs.open_file_dialog, ['<primary>N'], self)
         self.create_action('add_data_advanced', add_data_advanced.open_add_data_advanced_window, ['<primary><shift>N'], self)
         self.create_action('add_equation', add_equation.open_add_equation_window, ['<primary><alt>N'], self)
+        self.create_action('open_project', graphs.open_file_dialog, ['<shift><alt>N'], self, None, True)        
         self.create_action('select_all', graphs.select_all, ['<primary>A'], self)
         self.create_action('select_none', graphs.select_none, ['<primary><shift>A'], self)
         self.create_action('undo', item_operations.undo, ['<primary>Z'], self)
         self.create_action('redo', item_operations.redo, ['<primary><shift>Z'], self)
         self.create_action('restore_view', plotting_tools.restore_view, ['<primary>R'], self)
         self.create_action('view_back', plotting_tools.view_back, ['<alt>Z'], self)
         self.create_action('view_forward', plotting_tools.view_forward, ['<alt><shift>Z'], self)
-        self.create_action('export_figure', plotting_tools.export_figure, ["<primary>E"], self)
-        self.create_action('export_data', item_operations.export_data, ['<primary><shift>E'], self)
-        self.create_action('save_project', graphs.save_project_dialog, ['<primary>S'], self)
-        self.create_action('open_project', graphs.open_file_dialog, ['<primary>O'], self, None, True)
+        self.create_action('save_data', item_operations.save_data, ['<primary>S'], self)
```

Comment:
same hear, this breaks currently since the action called in window.ui is still the old one

Reply:
No good reasons for this, except that I was not working in a valid git directory and my branch was out of sync. Thanks for noticing.